tests/resources/UseCounters.conf

```diff
@@ -38,4 +38,3 @@
 
 method SVGSVGElement.getElementById
 attribute SVGSVGElement.currentScale
-property Fill
```

Comment:
`property` was removed from the list of keys allowed here, but I don't actually know what this line was doing before, and tests pass fine without it.